private/buf/cmd/buf/command/push/push.go

```diff
@@ -208,19 +208,16 @@ func run(
 			},
 		)
 	} else {
-		lines, err = slicesext.MapError(
+		if len(commits) > 1 {
+			linesErr = syserror.Newf("Received multiple commits back for a v1 module. We should only ever have created a single commit for a v1 module.")
+		}
+		lines = slicesext.Map(
 			commits,
 			// Printing dashless for historical reasons.
-			func(commit bufmodule.Commit) (string, error) {
+			func(commit bufmodule.Commit) string {
 				return uuidutil.ToDashless(commit.ModuleKey().CommitID())
 			},
 		)
-		if err != nil {
-			return err
-		}
-		if len(commits) > 1 {
```

Comment:
This shouldn't have been deleted

Reply:
It moved above - this wasn't dependent on the lines variable so I figured it could go above.

Reply:
ah, got it